modules/aws-eks-fargate-profiles/main.tf

```diff
@@ -1,19 +1,19 @@
 resource "aws_eks_fargate_profile" "eks_fargate" {
   cluster_name           = var.context.eks_cluster_id
-  fargate_profile_name   = "${local.fargate_profiles["fargate_profile_name"]}-${var.context.eks_cluster_id}"
```

Comment:
@vara-bonthu any issues here?

Reply:
In theory, users can set the same name scheme defined previously if this change were to go through. If today they set:
```hcl
  cluster_name = "bar"
  ...
  fargate_profile_name = "foo"
  ...
```

They can set the following to avoid replacement/disruption:
```hcl
  cluster_name = "bar"
  ...
  fargate_profile_name = "foo-bar"
  ...
```

Reply:
This was updated in my existing PR. Agree with the change